uniswap-v2/src/lib.rs

```diff
@@ -1,15 +1,20 @@
-use hex;
+use hex_literal::hex;
 use substreams::{log, proto, store};
 use substreams_ethereum::{Event as EventTrait, pb::eth::v2 as eth};
+use substreams_helper::{erc20, utils};
 
 use abi::factory;
+
+use pb::common;
+use pb::dex_amm;
 use pb::uniswap_v2;
-use substreams_helper::{erc20, utils};
 
 mod abi;
 mod pb;
 
-pub const UNISWAP_V2_FACTORY: &str = "5c69bee701ef814a2b6a3edd4b1652cb9cc5aa6f";
+type Address = [u8; 20];
```

Comment:
maybe we put `Address` and its utilities in substreams-helper/adddress.rs?

so that substreams devs could do `addresss!("5c69bee701ef814a2b6a3edd4b1652cb9cc5aa6f")` rather than understanding the usage of libs such as `hex_literal::hex` etc.

Think `import { Address } from "@graphprotocol/graph-ts", but for substreams.

Similarly, I am thinking we should have `bignumber.rs` in substreams-helper s.t. substreams devs don't have to deal with the complexity of num_bigint and bigdecimal on their own. Think `import { BigInt, BigDecimal } from "@graphprotocol/graph-ts". I probably do this shortly, wdyt?